MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.RecyclerView/Model/MvxViewHolderBindedEventArgs.cs

```diff
@@ -0,0 +1,18 @@
+namespace MvvmCross.Droid.Support.V7.RecyclerView.Model
+{
+    public class MvxViewHolderBindedEventArgs
+    {
+        public MvxViewHolderBindedEventArgs(int itemPosition, object dataContext, Android.Support.V7.Widget.RecyclerView.ViewHolder holder)
+        {
+            ItemPosition = itemPosition;
+            DataContext = dataContext;
+            Holder = holder;
+        }
+
+        public int ItemPosition { get; }
+
+        public object DataContext { get; }
+
+        public Android.Support.V7.Widget.RecyclerView.ViewHolder Holder { get; }
```

Comment:
ViewHolder constains all required informations like:
ItemView,
Item position,
ItemView Type.
I guess it's better to expose that than just pure View.